src/services/publishing/publishing.ts

```
@@ -13,7 +13,7 @@ const assertAddressIsSmartContract = async (address: string, wallet: Wallet): Pr
 export const publishVerifiableDocumentJob = async (
   job: PublishingJob,
   wallet: Wallet
-): Promise<string> => {
+): Promise<string | undefined> => {
```

should it throw an error instead if its undefined?